plugin/src/main/java/io/jenkins/plugins/casc/BaseConfigurator.java

```diff
@@ -143,7 +145,9 @@ protected Attribute detectActualType(String name, final TypePair type) {
         }
 
         Attribute attribute;
-        if (PersistedList.class.isAssignableFrom(type.rawType)) {
+        if (DescribableList.class.isAssignableFrom(type.rawType)) {
+            return new DescribableListAttribute(name, c);
```

Comment:
So now we can get a list of attributes to show all the configurable elements?

Reply:
![capture d ecran 2018-10-10 a 10 35 35](https://user-images.githubusercontent.com/132757/46723476-3f016d00-cc78-11e8-8868-17f54cedc776.png)

Reply:
🙌 and LINKS working too? 🙊

Reply:
yes, sounds like we reach some milestone here :) 🎉